src/pidgey/agama_backend.py

```diff
@@ -11,11 +11,18 @@ def __imports__():
 
         agama.setUnits(mass=1, length=1, velocity=1)
 
+    @property
     def ORBIT_TYPE(self):
-        return tuple
+        return np.ndarray
 
     def _compute_orbit(
-        self, skycoord, pot, dt, steps, pattern_speed=0 * u.km / u.s / u.kpc
+        self,
+        skycoord,
+        pot,
+        dt,
+        steps,
+        pattern_speed=0 * u.km / u.s / u.kpc,
```

Comment:
Now that I'm looking at this again: I wonder if `pattern_speed` makes more sense as an `integration_kwarg`? Promoting to a named argument makes sense for our bar-driven application, but since the tool is more flexible it might give a sense of importance that wasn't intended.

Reply:
This makes sense, but the pattern speed is currently used for `get_points` in some cases as well. Moving it into `integration_kwargs` and out of `get_points` might be better implemented in a future version.